cyences_app_for_splunk/default/data/ui/views/cs_overview.xml

```diff
@@ -226,7 +226,33 @@
       <title>Antivirus</title>
       <table>
         <search base="basesearch">
-          <query>| `cs_filter_savedsearches("Sophos|Windows Defender|CrowdStrike")`</query>
+          <query>| `cs_filter_savedsearches("Sophos|Windows Defender|CrowdStrike")` 
+| eval Name=if(Name="Sophos Service is not Running","Sophos Service Stopped/Disabled Events",Name) 
+| eval Name=if(Name="Sophos RealTime Protection Disabled","Sophos RealTime Protection Stopped/Disabled Events",Name) 
+| eval Name=if(Name="Endpoint Not Protected by Windows Defender","Windows Defender Endpoint Stopped/Disabled Events",Name) 
+| eval Name=if(Name="Windows Defender RealTime Protection Disabled or Failed","Windows Defender RealTime Protection Stopped/Disabled/Failed Events",Name) 
+| rename "Notable Events" as Count 
+| append 
+    [| search `cs_sophos` type IN ("Event::Endpoint::ServiceNotRunning", "Event::Endpoint::ServiceRestored") dhost IN ("*") user IN ("*") earliest=$timeRange.earliest$ latest=$timeRange.latest$
+    | fields _time host dhost type 
+    | stats latest(type) as type by host, dhost 
+    | where type=="Event::Endpoint::ServiceNotRunning" 
+    | stats count as Count
+    | eval Name="Currently Disabled/Stopped Sophos Services"] 
+| append
+    [| search `cs_sophos` type="Event::Endpoint::Sav*" host="*" dhost IN ("*") user IN ("*") earliest=$timeRange.earliest$ latest=$timeRange.latest$
+    | fields host dhost type 
+    | stats latest(type) as type by host, dhost 
+    | where type=="Event::Endpoint::SavDisabled" 
+    | stats count as Count 
+    | eval Name="Currently Stopped/Disabled Sophos RealTime Protection"]
+| append
+    [| search `cs_wineventlog_system` SourceName="Microsoft-Windows-Service Control Manager" Message="The Windows Defender Antivirus Service service entered the *"  earliest=$timeRange.earliest$ latest=$timeRange.latest$ | stats latest(Message) as Message by host | search Message=*stopped* | stats count as Count
+    | eval Name="Currently Disabled/Stopped Windows Defender Services"]
+| append
+    [| search `cs_windows_defender` EventCode=1151  earliest=$timeRange.earliest$ latest=$timeRange.latest$ | fields RTP_state host| stats latest(RTP_state) as RTP_state by host | where RTP_state=="Disabled" | stats count as Count
```

Comment:
I think each defender sends EventCode=1151 events every hour. So we can reduce the timerange to like last 4 hours only. We might need to do the same for the Health Report panel on the defender dashboard.
- But first, we need to confirm the fact that these events are being sent by all the hosts at regular intervals.

Reply:
use macro.

Reply:
This will not work. I checked the logs and for every 4 hours of span, we are not getting the same number of hosts. So, currently, we are not implementing this.